buildingmotif/dataclasses/library.py

```diff
@@ -342,6 +344,23 @@ def _load_from_directory(
 
         return lib
 
+    @classmethod
+    def load_from_libraries_yml(cls, filename: str):
```

Comment:
Because it refers to `Library`, this is tricky to do without introducing a circular dependency. I think it's ok as is, but we could consider moving it to some other package too

Reply:
@TShapinsky let me know what you think and I can make some adjustments

Reply:
@gtfierro the circular imports thing definitely makes sense.